src/components/item/index.tsx

```diff
@@ -20,10 +20,6 @@ export class Item {
 		this.selected = true
 		this.itemSelected.emit()
 	}
-	componentDidLoad() {
```

Comment:
Why was `smoothly-item` emitting `"itemSelected"` in the first place?
Also, why should it be removed now?